redis-hbo-provider/src/main/java/com/facebook/presto/statistic/RedisProviderPlugin.java

```diff
@@ -34,7 +34,6 @@ public class RedisProviderPlugin
 
     private Injector injector;
 
-    @VisibleForTesting
     public RedisProviderPlugin(String configPath)
```

Comment:
can we inline this code in the no argument constructor now.  It's the only thing that calls this.